### PR TITLE
Expeses: add description field to the expenses model

### PR DESCRIPTION
**Pull request by ronyyosef**
#51

### CODE REVIEW

File: expenses/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,38 +1,22 @@
...
     initial = True
 
-    dependencies = []
+    dependencies = [
+    ]
 
     operations = [
         migrations.CreateModel(
-            name="Expenses",
+            name='Expenses',
```
Comment (LiorKGOW) [inline expenses/migrations/0001_initial.py:15]: Is there a reason behind this change?
Is there a difference between " and ' in python in general? just making sure

**Review by LiorKGOW — APPROVED** [review]
LGTM
Dropped a comment but not something big

File: expenses/models.py [hunk 2]
```diff
@@ -15,6 +15,7 @@ class Category(models.TextChoices):
         CULTURE = 'Culture'
         VACATIONS = 'Vacations'
         OTHER = 'Other'
+    description = models.TextField(max_length=250, default='')
```
Comment (LiorKGOW) [inline expenses/models.py:18]: What is the puprpose of this field?
Adding extra text to the expense so the person can know what, where, on which ocasion etc.. ?
Just making sure

**Review by LiorKGOW — APPROVED** [review]

**Review by LiorKGOW — APPROVED** [review]

File: expenses/migrations/0001_initial.py (hunk 1 above)
Comment (ronyyosef) [reply expenses/migrations/0001_initial.py:15]: There isn't a really big difference
you can go and read a bit but it is mostly a code-style
https://stackoverflow.com/questions/56011/single-quotes-vs-double-quotes-in-python
It was created like it by the Django framework and I didn't touch it

File: expenses/migrations/0001_initial.py [hunk 3]
```diff
@@ -1,38 +1,22 @@
...
-                            ("Vacations", "Vacations"),
-                            ("Other", "Other"),
-                        ],
-                        default="Other",
-                        max_length=32,
-                    ),
-                ),
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('description', models.TextField(default='', max_length=250)),
+                ('category', models.CharField(choices=[('Rent', 'Rent'), ('Mortgage', 'Mortgage'), ('bills', 'Bills'), ('Transportation', 'Transportation'), ('Clothing', 'Clothing'), ('Healthcare', 'Healthcare'), ('Food', 'Food'), ('Insurance', 'Insurance'), ('Kids', 'Kids'), ('Culture', 'Culture'), ('Vacations', 'Vacations'), ('Other', 'Other')], default='Other', max_length=32)),
```
[the remark this answers is not in the document]
Comment (ronyyosef) [reply expenses/migrations/0001_initial.py:19]: I fixed the flake8 to ignore the migrations file
take a look at the Github action of the flake8

**Review by LiorKGOW — APPROVED** [review]

**Comment by ronyyosef** [comment]
Hey @adiabramovitch 
there are 2 commits in this PR
The first one was actually add of the description to the expenses model
The second one is to ignore long lines that generated by the Django framework
If necessary I can split it into two PRs, merge the flake8 fix, and after that one merge actually added the description
I don't really see a reason to split these two

**Comment by adiabramovitch** [comment]
@ronyyosef  please follow https://cbea.ms/git-commit/ the commit-msg is supposed to start with uppercase and then I can merge it

**Comment by ronyyosef** [comment]
Fixed!

**Review by Yoavsc129 — APPROVED** [review]
Everything looks fine

**Comment by adiabramovitch** [comment]
@ronyyosef resolve conflicts and I can merge

File: house/tests/house_tests.py [hunk 4]
```diff
@@ -37,7 +37,7 @@ def generate_house(generate_country, generate_city):
 
 
 @pytest.mark.django_db
-class TestCourseModel:
+class TestHouseModel:
```
Comment (shlior7) [inline house/tests/house_tests.py:40]: Im pretty sure this change and most of the changes in this commit were already merged so this needs to be rebased

**Comment by ronyyosef** [comment]
This PR is ready to merge

**Pull request merged**